container/webui/nginx.conf

```diff
@@ -30,7 +30,7 @@ http {
             proxy_set_header Connection $connection_upgrade;
             proxy_set_header Host $host;
 
-            rewrite /api/v1/ws/(.*) /ws/$1 break;
+            rewrite /api/v1/ws/(.*) /api/v1/ws/$1 break;
```

Comment:
I understand your reasoning in the commit message but to me this looks like `/api/v1/ws/<foo>` is "rewritten" to `/api/v1/ws/<foo>` meaning not rewritten at all. What am I overlooking?

Reply:
Yes, isn't the idea to be able to drop the rewrite completely?

Reply:
Not rewritten at all, that is the intended behavior (or to put it differently, the rewrite is no longer needed). I could probably delete the rewrite completely, but I am not sure if it has another side-effect, as the same style of rewrite can be found in the liveviewhandler section: `rewrite /liveviewhandler/(.*) /liveviewhandler/$1 break;`.

If you are sure, it has no other functionality (like disabling other possible rewrites due to the `break`) I would delete both of them to keep it consistent.

Reply:
I suppose both rewrites can be deleted then.